packages/terra-avatar/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed 
+* Changed generateImage function to Fallback to Initials if Provided.
```

Comment:
Maybe saying 'Changed the load behavior such that if the image fails to load and initials are provided, the avatar falls back to the initials display.' is more clear to a consumer.

Reply:
Updated [here](https://github.com/cerner/terra-core/blob/565f1417aaeb24a8bcc372b80c65ba24ebe360ab/packages/terra-avatar/CHANGELOG.md)